display_list/display_list_builder.h

```diff
@@ -640,6 +622,7 @@ class DisplayListBuilder final : public virtual Dispatcher,
   DlPaint current_;
   // If |current_blender_| is set then ignore |current_.getBlendMode()|
   sk_sp<SkBlender> current_blender_;
+  DisplayListMatrixClipTracker tracker_;
```

Comment:
Maybe locate this near the current_layer_/layer_stack_ declarations up above (around line 500)?

Reply:
Done.l